jsconfig.json

```diff
@@ -3,7 +3,9 @@
     "baseUrl": "src",
     "paths": {
       "common": ["src/common/*"],
-      "utils": ["src/utils/*"]
+      "utils": ["src/utils/*"],
+      "ui-kit": ["src/ui-kit/*"],
+      "scss-kit": ["src/ui-kit/*"]
```

Comment:
why do we need to duplicate this ?????????????????????????????????????????? they point to the same  path - src/ui-kit/*